ui/src/app/utils/authentication.tsx

```diff
@@ -87,6 +89,7 @@ export const signIn = (): void => {
 
 export const signOut = async (continuePath: string = '/login') => {
   authStore.set({ ...authStore.get(), isSignedIn: false });
+  sessionStorage.removeItem(DEMOGRAPHIC_SURVEY_SESSION_KEY);
```

Comment:
I should let session do its own thing, but by doing so this will let users redirect to survey on every login

Reply:
I'm not following - could you go into more detail about how this line changes the behavior vs not including it?

Reply:
Could you go into more detail about how behavior changes when this line is here vs. when it is removed?  Also what is the behavior when the user closes their browser instead of signing out?